content/en/privacy.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: Privacy Policy
 slug: privacy
-lastmod: 2023-08-23
+lastmod: 2024-06-26
 english_is_canonical: 1
 show_lastmod: 1
 ---
 
 The Let's Encrypt Privacy Policy describes how we collect, use, and disclose your information in three different contexts:
 
-- When, as a Relying Party, you visit a web site secured with HTTPS that uses a certificate from Let's Encrypt,
+- When, as a Relying Party, you vi`sit a web site secured with HTTPS that uses a certificate from Let's Encrypt,
```

Comment:
```suggestion
- When, as a Relying Party, you visit a web site secured with HTTPS that uses a certificate from Let's Encrypt,
```

Reply:
Added in a8a9366

Reply:
I don't see a diff on this line in a8a9366, and it looks like the stray backtick is still present in the version that got merged?